index.html

```diff
@@ -62,8 +62,7 @@
 <script src="http://js.arcgis.com/4.10/"></script>
 
 <!--reference the javascript for statistics spatial query-->
-<script src="querttest.js"></script>
-<script src="query.js"></script>
```

Comment:
The `Query.js` file still contains some things that must be moved over to the new file `QueryLib.js`.